crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -301,6 +282,20 @@ impl EntityGeneration {
 ///
 /// [generational index]: https://lucassardois.medium.com/generational-indices-guide-8e3c5f7fd594
 ///
+/// # Entity ids
```

Comment:
This section of the docs is really important to get right, but I don't think it lands in its current form. Ultimately it feels like this should be in the module docs, and start with a better description of the lifecycle of entities. I kind of see where you're going for with "conceptual entity", but I really think it's going to badly confuse people.